subiquitycore/controllers/network.py

```diff
@@ -237,6 +238,21 @@ def netplan_path(self):
             netplan_config_file_name = '00-snapd-config.yaml'
         return os.path.join(self.root, 'etc/netplan', netplan_config_file_name)
 
+    def add_vlan(self, device, vlan):
```

Comment:
I'd like to think of a way to do this where dry-run doesn't have to run as root. But this is probably OK for this branch.

Reply:
https://trello.com/c/2Opt8urP